### ISSUE TITLE
Ability to chain api calls

### ISSUE THREAD

**Issue by mwq27** [octokit/octokit.js#195]
Is it possible to chain api calls like so - 

```
var git = new GitHubApi({
   ...
});

var srcGit = new GitHubApi({
    ...
});

srcGit.issues.repoIssues({
...}, function(err, res){ 
     git.issues.create({
                ...
            }, function(err, res){
    srcGit.issues.getComment({ ... }, function(err, res){
    git.issues.createComment({ ... }, function(err, res){});
});
});
});
```

I'm trying to move issues from one repo to another.  Creating the issues is working, but trying to add the comments to each new issue im running into problems, since the for loops seems to continue while the API call is still working.

**Comment by kaizensoze** [comment]
You'll want to use promises. To promisify node-github, see here: http://bluebirdjs.com/docs/api/promise.promisifyall.html

**Comment by kaizensoze** [comment]
Actually, the ability to use promises will be included in the next release. See [example](https://github.com/mikedeboer/node-github/blob/master/examples/testPromise.js).

**Comment by kaizensoze** [comment]
Included in latest 2.1.0.
